webapp/channels/src/components/user_settings/notifications/user_settings_notifications.tsx

```diff
@@ -284,7 +287,20 @@ class NotificationsTab extends React.PureComponent<Props, State> {
         this.setState({isSaving: true});
         stopTryNotificationRing();
 
-        const {data: updatedUser, error} = await this.props.updateMe({notify_props: data});
+        let updatedUser: UserProfile | undefined;
+        let error;
+
+        if (this.props.adminMode) {
+            const payloadUser = {...this.props.user, notify_props: data};
+            const response = await this.props.patchUser(payloadUser);
```

Comment:
(1/5) Since this is a patch, can we just pass the `{notify_props: ...}` without the rest of those fields? Passing the whole object means an admin could potentially overwrite something that they don't mean to or that isn't loaded by them

Reply:
We should, but everywhere in webapp are passing the whole user profile object and the types also state that it should be a full user. Will have to test the API and make some changes in webapp. So skipping this for now.